g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/TranslationProductAction.java

```diff
@@ -190,8 +186,7 @@ public APIResponseDTO getPartialComTrans(String productName,
                HttpServletRequest req) throws Exception {
           
           TranslationDTO resulttranslationDTO = getResultTranslationDTO( productName, version,components, locales,  pseudo, req);
-          TranslationDTO allTranslationDTO  =  getAllCompTrans( productName,  version, pseudo, req);
-        
+          TranslationDTO allTranslationDTO  =  getAllCompTrans( resulttranslationDTO.getProductName(),  resulttranslationDTO.getVersion(), pseudo, req);
```

Comment:
Should pseudo also from resulttranslationDTO? It's better to aligh with parameters 'product' and 'version'.

Reply:
done